com.unity.render-pipelines.high-definition/Editor/RayTracing/PlanarReflectionAngleGenerator.cs

```diff
@@ -0,0 +1,143 @@
+using System.IO;
+
+namespace UnityEngine.Rendering.HighDefinition
+{
+    internal class PlanarReflectionAngleGenerator : MonoBehaviour
+    {
+        void Start()
```

Comment:
Don't leave the empty `Start` method, it is costly even if it is empty.

Reply:
it doesn't matter, this is an offline script to generate a table